index.html

```diff
@@ -0,0 +1,231 @@
+<!Doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey Form </title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <header class="header">
+          <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
+          <p id="description" class="description text-center">
+            Thank you for taking the time to help us improve the platform
+          </p>
+        </header>
+        <form id="survey-form">
+          <div class="form-group">
+            <label id="name-label" for="name">Name</label>
```

Comment:
Please  include this part in <ul><li> tag because it is important use the appropriate semantic tag .